src/Tizen.NUI.Components/Controls/RecyclerView/RecycleLayoutManager.cs

```diff
@@ -101,6 +92,13 @@ public enum Orientation
         [EditorBrowsable(EditorBrowsableState.Never)]
         public int DataCount{get; set;}
 
+        /// <summary>
+        /// FIXME
```

Comment:
The last scrolled position which is calculated by ScrollableBase. The value should be updated in Recycle() method.